entities/config.go

```diff
@@ -12,29 +12,21 @@ type Client struct {
 	ID    string `toml:"user_id"`
 }
 
-type ServerConfig_t struct {
-	MotherShipIP   string            `toml:"motherShipIP"`
-	MotherShipPort string            `toml:"motherShipPort"`
-	MotherShipSSL  bool              `toml:"motherShipSSL"`
+type Server struct {
```

Comment:
Where can I set port or ip

Reply:
They both are set with flags when running the server

Reply:
If no flag provided then its localhost and 7878 if yes then okay. But I am not setting flags for local testing

Reply:
yep they both have default values